### PR TITLE
Merge the native and kubernetes quickstart in getting started

### PR DESCRIPTION
**Pull request by cescoffier**
Because the project generated by the user contains everything required, no need for 3 quickstarts anymore. 

Also, this PR make the instructions more consistent.

### CODE REVIEW

File: getting-started/src/main/resources/META-INF/microprofile-config.properties [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Configuration file
+key = value
```
Comment (gsmet) [inline getting-started/src/main/resources/META-INF/microprofile-config.properties]: This file should be removed.

Comment (cescoffier) [reply getting-started/src/main/resources/META-INF/microprofile-config.properties]: File deleted.

**Comment by cescoffier** [comment]
Yes, this PR is against `master` on purpose. It's an improvement of the guide that should be publish soon (without a release)

**Pull request merged**